### PR TITLE
User profile and address schema

### PR DESCRIPTION
**Pull request by maysieo**
Just made a pull request to dev. It's a chonky boy--my apologies. I hadn't realized how unruly it had become. 

- Shows the user profile up and running. Currently there are working tabs for previous and future booking, and stubbed-out tabs for Favorite Vendors and Estates.
    - There's still a lot of styling I plan/need to do.
    - I also need to swap out the nav bar.
- There's also a new db schema to allow easier storage and access to addresses
  - There's also a corresponding seed file that seeds 200 addresses--so that hopefully most of our users have more than one address that we can play around with.

### CODE REVIEW

File: manorMatch/src/components/UserProfile/UserProfile.jsx [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+    .catch((error) => {
+      console.error('Error:', error);
+    })
+}
+
+  useEffect(() => {
+    const getUserAndBookings = () => {
+      const apiUrl = import.meta.env.VITE_API_URL;
+      const userEmail = 'Naomie_Fadel65@yahoo.com';
```
Comment (amarinsam) [inline manorMatch/src/components/UserProfile/UserProfile.jsx:56]: Testing!  Here!

**Comment by amarinsam** [comment]
The den is so cool!  Just added a comment to one line in the 'files changed' tab!

**Review by amarinsam — CHANGES_REQUESTED** [review]
Naomie Fadel!

File: manorMatch/src/components/UserProfile/UserProfile.jsx (hunk 1 above)
Comment (maysieo) [reply manorMatch/src/components/UserProfile/UserProfile.jsx]: Hard coding: out! Getting from local storage: in!

**Comment by amarinsam** [comment]
SO cool thanks Maysie!

**Review by amarinsam — APPROVED** [review]
Looks awesome, thanks!!

**Pull request merged**